Kitodo/src/main/java/org/kitodo/production/services/workflow/WorkflowControllerService.java

```diff
@@ -101,10 +112,29 @@ public void setTaskStatusUp(Task task) throws DataException, IOException {
      *            to change status down
      */
     public void setTaskStatusDown(Task task) {
-        task.setEditType(TaskEditType.ADMIN);
-        task.setProcessingTime(new Date());
-        taskService.replaceProcessingUser(task, getCurrentUser());
-        setProcessingStatusDown(task);
+        setTaskStatusDown(Arrays.asList());
```

Comment:
The parameter `task` is never used. Should it be passed to `setTaskStatusDown` here?